apps/judicial-system/web/src/utils/validate.ts

```diff
@@ -204,10 +204,12 @@ export const isDefendantStepValidIndictments = (
 ) => {
   const result =
     policeCaseNumbers.length > 0 &&
+    workingCase.indictmentSubTypes &&
+    Object.entries(workingCase.indictmentSubTypes).length > 0 &&
     !someDefendantIsInvalid(workingCase) &&
     validate([
       [workingCase.type, ['empty']],
-      [workingCase.indictmentSubType, ['empty']],
+      [Object.entries(workingCase.indictmentSubTypes)[0][1][0], ['empty']],
```

Comment:
This is unclear to me. Please consider writing a helper function for this and explain the [0][1][0] thing with a comment

Reply:
This is throwaway code, so I would rather not do that. The whole logic will change when the client implements multiple indictment sub-types.